Button/story.jsx

```diff
@@ -40,6 +40,24 @@ storiesOf('Button')
       </Button>
     </div>
   ))
+  .add('hovered', () => (
+    <div>
+      <p>Default</p>
+      <Button hovered>Button text</Button><br /><br />
+      <p>Borderless</p>
+      <Button hovered borderless>Button text</Button><br /><br />
+      <p>Large</p>
+      <Button hovered large>Button text</Button><br /><br />
+      <p>Secondary</p>
+      <Button hovered secondary>Button text</Button><br /><br />
+      <p>Small</p>
+      <Button hovered small>Button text</Button><br /><br />
+      <p>Tertiary</p>
+      <Button hovered tertiary>Button text</Button><br /><br />
+      <p>Warning</p>
+      <Button hovered warning>Button text</Button>
+    </div>
+  ))
```

Comment:
Love even from the implementation side of this, from the get-go we can see right away the state of `Button`. I think this might make targeting specific states perhaps a lot more easier to read and implement, too.